core/dnsserver/config.go

```diff
@@ -68,16 +69,22 @@ func (c *Config) HostAddresses() string {
 	return all
 }
 
+// build a key for identifying the configs during setup time
```

Comment:
This isn't a proper Go comments, start iwth `keyForConfig ...`

Reply:
ok. update